plugins/source/github/resources/services/repositories/alerts.go

```diff
@@ -30,8 +29,8 @@ func fetchAlerts(ctx context.Context, meta schema.ClientMeta, parent *schema.Res
 			return err
 		}
 		res <- alerts
-		opts.Page = strconv.FormatInt(int64(resp.NextPage), 10)
-		if resp.NextPage == 0 {
+		opts.After = resp.After
+		if len(resp.After) == 0 {
```

Comment:
LGTM as this is the same pattern we use in `resources/services/organizations/alerts.go`